src/test/java/test/IngressTest.java

```diff
@@ -222,7 +222,103 @@ void jira_ingress_path_default(Product product) throws Exception {
         assertThat(ingresses.head().getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path"))
                 .hasTextEqualTo("/");
     }
+    
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
+    void bamboo_ingress_host_port(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.host", "myhost.mydomain",
+                "ingress.port", "666"));
+
+        resources.getStatefulSet(product.getHelmReleaseName()).getContainer().getEnv()
+                .assertHasValue("ATL_PROXY_NAME", "myhost.mydomain")
+                .assertHasValue("ATL_PROXY_PORT", "666");
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
+    void bamboo_ingress_port(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.host", "myhost.mydomain"));
+
+        resources.getStatefulSet(product.getHelmReleaseName()).getContainer().getEnv()
+                .assertHasValue("ATL_PROXY_NAME", "myhost.mydomain")
+                .assertHasValue("ATL_PROXY_PORT", "443");
+    }
 
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
+    void bamboo_ingress_port_http(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.host", "myhost.mydomain",
+                "ingress.https", "False"));
+
+        resources.getStatefulSet(product.getHelmReleaseName()).getContainer().getEnv()
+                .assertHasValue("ATL_PROXY_NAME", "myhost.mydomain")
+                .assertHasValue("ATL_PROXY_PORT", "80");
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
+    void bamboo_ingress_path_contextPath(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain",
+                "bamboo.service.contextPath", "/bamboo-tmp"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertEquals(1, ingresses.size());
+
+        assertThat(ingresses.head().getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path"))
+                .hasTextEqualTo("/bamboo-tmp");
+
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "bamboo")
+    void bamboo_ingress_path_value(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain",
+                "bamboo.service.contextPath", "/context",
```

Comment:
I was recently thinking about that. Is there a real use case to have `ingresPath` and `contextPath` different? I didn't implement this use case for Bitbucket helm chart in https://github.com/atlassian/data-center-helm-charts/pull/314 because I am not sure if there is a need for it. Is that different for Bamboo?

Reply:
Its no different for bamboo, but I noted confluence was testing for it, so in the interests of flexibility I thought making sure had the flexibility too. To be honest Im not sure if this is a real use case but its not exactly like we can stop people from providing different values for these properties in `values.yaml`?